alibi_detect/saving/tests/models.py

```diff
@@ -89,7 +89,7 @@ def encoder_dropout_model(backend, current_cases):
 
 
 @fixture
-def preprocess_custom(encoder_model):
+def preprocess_uae(encoder_model):
```

Comment:
More descriptive naming. `preprocess_custom` actually contains a uae model, so renamed `preprocess_uae`. Otherwise confusion in `test_saving.py` where we are now testing "custom" preprocessing functions.